pyerrors/input/sfcf.py

```diff
@@ -6,17 +6,41 @@
 import re
 import numpy as np  # Thinly-wrapped numpy
 from ..obs import Obs
+from . import utils
 
-
-def read_sfcf(path, prefix, name, **kwargs):
-    """Read sfcf C format from given folder structure.
+def read_sfcf(path, prefix, name, quarks='.*', noffset=0, wf=0, wf2=0, version = "1.0c", **kwargs):
+    """Read sfcf c format from given folder structure.
```

Comment:
Documentation of the parameters `path`, `prefix` and `name` seems to be missing.